base/applications/regedit/framewnd.c

```diff
@@ -1230,7 +1230,14 @@ static BOOL _CmdWndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
         FindDialog(hWnd);
         break;
     case ID_EDIT_FINDNEXT:
-        FindNext(hWnd);
+        if (!FindNext(hWnd))
```

Comment:
This basically duplicates the existing code. I think you should refactor it into a separate function.

https://github.com/reactos/reactos/blob/6aae5f4fdd87efe805f7bd44c601044b0cbe47c2/base/applications/regedit/find.c#L818

Reply:
Done. Thanks @binarymaster. :+1: 
Still works for me without any issues.

Reply:
But why is then the function used only once?

Reply:
Why I need call it two or more times @HBelusca? What this will improve? It works for me as-is, the behaviour is the same as in 2k3 Regedit. What's wrong? :confused:

Reply:
I understood @binarymaster 's comment as the fact that this code chunk was duplicated at least two times, hence needing to be placed instead in a function in order to remove the code duplication. But if that code chunk was actually present just once, I don't see why moving it inside a function would help.

Reply:
I see what you mean, it's possible to move message box creation into `FindNext()`, but I think in current condition of this PR the code is more readable.